packages/zoe/tools/scriptedOracle.js

```diff
@@ -10,12 +11,11 @@ import { E } from '@agoric/eventual-send';
  * result, but has no impact on the result. If the script has no entry for the
  * time, the event is 'nothing to report'.
  *
- * @param {Array} script
+ * @param {Record<string, any>} script
  * @param {Installation} oracleInstallation
- * @param {Timer} timer
+ * @param {TimerService} timer
```

Comment:
Were the types of `script` and `timer` previously incorrect but we didn't notice because of the lack of `@ts-check`?

Reply:
Not sure, I think this is also a @michaelfig change